README.md

```diff
@@ -1,42 +1,36 @@
-# Knative Client
+# Kn
 
-This section outlines best practices for the Knative developer experience. It is a reference for Knative CLI implementation, and a reference for Knative client libraries.
+The Knative client `kn` is your door to the [Knative](https://knative.dev) world.
+It allows you to create Knative resources interactively from the command line or from within Shell scripts.
 
-The goals of the Knative Client are to:
+`kn` offers you ...
 
-1. Follow the Knative [serving](https://github.com/knative/serving) and [eventing](https://github.com/knative/eventing) APIs
-2. Be scriptable to allow users to create different Knative workflows
-3. Expose useful Golang packages to allow integration into other programs or CLIs or plugins
-4. Use consistent verbs, nouns, and flags for various commands
-5. Be easily extended via a plugin mechanism (similar to `kubectl`) to allow for experimentation and customization
+* ... full support for managing all features of [Knative Serving](https://github.com/knative/serving) (services, revisions, traffic splits)
+* ... growing support [Knative eventing](https://github.com/knative/eventing), closely following its development (managing of sources & triggers)
+* ... a plugin architecture similar to that of `kubectl` plugins
+* ... a thin client-specific API in golang which helps in tasks like synchronously waiting on Knative service write operations.
+* ... easy integration of Knative into Tekton Pipelines by using `kn` in a Tekton `Task`.
```

Comment:
add a link for task https://github.com/tektoncd/catalog/tree/master/kn ?

Reply:
Not too sure about the `...` at start of the list. Not clear that makes it easier to harder to read. Anyhow my .02